compiler+runtime/include/cpp/jank/analyze/expr/case.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <jank/detail/to_runtime_data.hpp>
+#include <jank/analyze/expression_base.hpp>
+
+namespace jank::analyze::expr
+{
+  using namespace jank::runtime;
+
+  template <typename E>
+  struct case_ : expression_base
+  {
+    native_box<E> value_expr{};
+    native_integer shift{};
+    native_integer mask{};
+    native_box<E> default_expr{};
+    std::vector<native_integer> transformed_keys{};
```

Comment:
Why is this called `transformed_keys`? They are not being transformed, from what I've seen. Is it because they're ints/hashes of keys? Either way, I think this should just be called `keys`.

Reply:
right. i've updated it.